README.md

```diff
@@ -126,47 +88,84 @@ kustomize build 'github.com/kubernetes-sigs/descheduler/kubernetes/deployment?re
 
 See the [user guide](docs/user-guide.md) in the `/docs` directory.
 
-## Policy and Strategies
+## Policy, Default Evictor and Strategy plugins
+
+**⚠️ v1alpha1 configuration is still suported, but deprecated (and soon will be removed). Please consider migrating to v1alpha2 (described bellow). For previous v1alpha1 documentation go to docs/deprecated/v1alpha1.md**
+
+The Descheduler Policy is configurable and includes default eviction strategy plugins that can be enabled or disabled. It includes a common eviction configuration at the top level, as well as configuration from the Evictor plugin (Default Evictor, if not specified otherwise). Top-level configuration and Evictor plugin configuration are applied to all evictions.
```

Comment:
There's only one default eviction strategy/plugin. By "default eviction strategy plugins", are you referring to various params of the default evictor plugin? Or, are you providing a generic description for custom eviction plugins which might eventually become default plugins?

Reply:
Eviction strategy plugins to me are the stretagies themselves. 

Evictor plugins are the ones configuring filters and sorters and etc (and we currently only have the default evictor as this type)

Maybe what I am calling Eviction strategy plugins we can call just strategy plugins? We need to call them something else than just plugins.

Reply:
Descheduling plugins/strategies. Eviction plugins corresponds too much with implementation of the Evict extension point.